cafebabel/templates/articles/edit.html

```diff
@@ -67,7 +67,7 @@ <h1 class="edit {% if article.id %}active{% endif %}">
     {% endif %}
   </div>
   <div class=textarea>
-    <label for=summary>Summary</label>
+    <label for=summary>Teaser</label>
```

Comment:
What about translations and tags which have the same label?

Reply:
Translations won't have the same _Teaser_. Never. And for the tags, I think we said we don't care if tags are spelled the same. MeetMtHood in english won't have nothing to do with MeetMyHood in french. They both exists and have a list of articles, that's all. Right ?

Reply:
For tags I get it but for translations you will keep Summary within the form and it will be inconsistent with that one (Teaser)

Reply:
should I change the `for=summary` to `for=teaser`?

Reply:
I suppose that's a good question that will occur in multiple places. We might need a namespace that would make a context. But that will be when translating I suppose.

Reply:
@cafebabel-webmaster no, I mean [here](https://github.com/cafebabel/cafebabel.com/blob/master/cafebabel/templates/articles/translations/edit.html#L17) and [there](https://github.com/cafebabel/cafebabel.com/blob/master/cafebabel/templates/articles/translations/create.html#L19) it will create inconsistency in the naming.

Reply:
Ok, in other word here for an article we call it "Teaser", but [for a translation we call it "Summary"](https://github.com/cafebabel/cafebabel.com/blob/master/cafebabel/templates/articles/translations/edit.html#L17).
The question is why do we call it different here, or if they all should be called "Teaser", or this one should remain "Summary".

Reply:
done: https://github.com/cafebabel/cafebabel.com/pull/220/commits/ee76d5d98f6eee4f2af284c9157c206fad47b4a3, https://github.com/cafebabel/cafebabel.com/pull/220/commits/1156dab3d50e84989a5bb2bb13f1975655722fb2